Soomla/domain/CCLeaderboard.h

```diff
@@ -14,8 +25,10 @@
 
 namespace soomla {
     class CCLeaderboard : public CCDomain {
```

Comment:
@borisspektor, to unify coding approach (CCUserProfile has such inheritance model) and to use this class https://github.com/soomla/soomla-cocos2dx-core/blob/master/Soomla/CCDomainFactory.h for easy serialization/deserialization.